src/main/java/graphql/normalized/incremental/NormalizedDeferExecution.java

```diff
@@ -103,11 +103,11 @@
  *
  */
 @ExperimentalApi
-public class DeferExecution {
```

Comment:
Can we just make this another small PR? I think this (and maybe other stuff) is just a refactoring, right?